.github/dependabot.yml

```diff
@@ -16,6 +16,11 @@ updates:
           interval: "daily"
       allow:
           - dependency-type: "development"
+      ignore:
+        - dependency-name: "phpstan/*"
+        - dependency-name: "phpunit/phpunit"
+          versions: [ ">= 9.0.0" ]
```

Comment:
Yes, we probably don't want the bot to try a major update of PHPUnit, which would be incompatible.